### ISSUE TITLE
pot output for plurals mismatches specification

### ISSUE THREAD

**Issue by colindean** [xitrum-framework/scala-xgettext#12]
I get this:

```
msgid "Waiting for {0} in-flight document to finish processing..."
msgid_plural "Waiting for {0} in-flight documents to finish processing..."
msgstr[0] ""
```

when I expect something like

```
msgid "Waiting for {0} in-flight document to finish processing..."
msgid_plural "Waiting for {0} in-flight documents to finish processing..."
msgstr[0] ""
msgstr[1] ""
```

https://www.gnu.org/software/gettext/manual/gettext.html#Translating-plural-forms

**Comment by colindean** [comment]
It looks like the _minimal_ change for my use case is to simply add `msgstr[1]` [here](https://github.com/xitrum-framework/scala-xgettext/blob/master/src/main/scala/scala/Xgettext.scala#L182
):

```scala
            msgidPluralo.map { msgidPlural =>
              builder.append(s"msgid_plural ${msgidPlural}\n")
              builder.append("msgstr[0] \"\"" + "\n\n") // ADD IT AFTER THIS
            }.getOrElse(builder.append("msgstr \"\"" + "\n\n"))
```

**Comment by colindean** [comment]
I've got a fix ready for this, but I have to go through my company's legal team before I can submit it. I expect that to take about a week 😒

**Comment by ngocdaothanh** [comment]
This issue is related to #13.

There are 2 reasons using only one entry `msgstr[0]` is OK:

* The generated pot file is supposed to be a template file. You copy it to create various language po files, `nplurals` for of the languages can be 1, 2, 3 etc. So keeping things to minimal is desirable (one entry `msgstr[0]` and the [pot header](https://github.com/xitrum-framework/scala-xgettext/blob/master/src/main/scala/scala/Xgettext.scala#L34
) doesn't include `nplurals`).
* Tools like Poedit can handle `nplurals` correctly. It can dynamically add and remove entries when you change `nplurals`.

**Comment by colindean** [comment]
I've submitted #16 for #13. I'm going to wait until that's merged so I can rebase this work on top of that.

I've pushed back on our translation team a little and asked them to implement a behavior similar to Poedit's. No response yet.

**Comment by colindean** [comment]
Meh, I pushed up the code anyway.

https://github.com/xitrum-framework/scala-xgettext/compare/master...colindean:colindean/adds-msgstr1-gh12

**Comment by ngocdaothanh** [comment]
I've merged your PR #16. Thanks.

**Comment by ngocdaothanh** [comment]
Close this issue as I think it has been addressed.
